diff --git a/public/controllers/store_controller.js b/public/controllers/store_controller.js
index 51c6149..d2f7152 100644
--- a/public/controllers/store_controller.js
+++ b/public/controllers/store_controller.js
@@ -1,11 +1,15 @@
 import { Controller } from 'stimulus';
+import axios from 'axios';
 
 export default class extends Controller {
     static get targets() {
         return ['productList'];
     }
 
-    printProducts() {
-        this.productListTarget.innerHTML = 'Testing!'
+    async printProducts() {
+        const response = await axios.get('/products');
+        const products = response.data;
+
+        this.productListTarget.innerHTML = products.join(', ');
     }
 }
