diff --git a/assets/js/app.js b/assets/js/app.js
index e8e3e00..04f5120 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -4,16 +4,17 @@ import Store from './components/store';
 import axios from 'axios';
 
 const storeEl = document.querySelector('#store');
+if (storeEl) {
+    async function startStoreApp() {
+        const response = await axios.get('/api/products');
+        const products = response.data.products;
 
-async function startStoreApp() {
-    const response = await axios.get('/api/products');
-    const products = response.data.products;
+        const store = new Store(storeEl, products);
+        store.initialize();
+    }
 
-    const store = new Store(storeEl, products);
-    store.initialize();
+    startStoreApp();
 }
 
-startStoreApp();
-
 const h1 = document.querySelector('h1');
 h1.innerHTML = `<img src="${sheep}" alt="baa" height="30">` + h1.innerHTML;
