diff --git a/assets/js/app.js b/assets/js/app.js
index 21d66c1..553f05b 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -5,11 +5,15 @@ import axios from 'axios';
 
 const storeEl = document.querySelector('#store');
 
-const response = await axios.get('/api/products');
-const products = response.data.products;
+function startStoreApp() {
+    const response = await axios.get('/api/products');
+    const products = response.data.products;
 
-const store = new Store(storeEl, products);
-store.initialize();
+    const store = new Store(storeEl, products);
+    store.initialize();
+}
+
+startStoreApp();
 
 const h1 = document.querySelector('h1');
 h1.innerHTML = `<img src="${sheep}" alt="baa" height="30">` + h1.innerHTML;
