diff --git a/templates/base.html.twig b/templates/base.html.twig
index f21fe81..3b8b584 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -10,6 +10,6 @@
             {% block body %}{% endblock %}
         </div>
         <script src="https://kit.fontawesome.com/a20a0555e5.js" crossorigin="anonymous"></script>
-        <script src="/app.js"></script>
+        <script src="/build/app.js"></script>
     </body>
 </html>
