diff --git a/templates/base.html.twig b/templates/base.html.twig
index fe78e19..da9fb4b 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -5,6 +5,7 @@
         <title>UK Symfony Meetup: Webpack Encore & A Modern JS Update</title>
         {% block stylesheets %}
             <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+            {{ encore_entry_link_tags('app') }}
         {% endblock %}
     </head>
     <body>
