diff --git a/public/app.js b/public/app.js
index cb43aae..36bfe05 100644
--- a/public/app.js
+++ b/public/app.js
@@ -1,5 +1,8 @@
 import Store from './store';
 
+// Could this somehow load a CSS file for us?
+import './styles.css';
+
 const app = document.querySelector('#app');
 const store = new Store(app, [
     'Sheer Shears',
diff --git a/public/styles.css b/public/styles.css
new file mode 100644
index 0000000..880b4e8
--- /dev/null
+++ b/public/styles.css
@@ -0,0 +1,6 @@
+#app {
+    background: green;
+    border-radius: 5px;
+    padding: 5px;
+    color: white;
+}
