diff --git a/webpack.config.js b/webpack.config.js
index 6d518c3..44e781b 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,25 +1,12 @@
-const path = require('path');
+var Encore = require('@symfony/webpack-encore');
 
-// webpack.config.js
-module.exports = {
-    entry: {
-        app: './public/app.js'
-    },
-    output: {
-        path: path.resolve(__dirname, 'public', 'build'),
-    },
-    mode: 'development',
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                exclude: /node_modules/,
-                loader: 'babel-loader'
-            },
-            {
-                test: /\.css$/,
-                use: ['style-loader', 'css-loader']
-            }
-        ]
-    }
-}
+Encore
+    .setOutputPath('public/build/')
+    .setPublicPath('/build')
+
+    // will output as public/build/app.js
+    .addEntry('app', './public/app.js')
+    .enableSourceMaps(!Encore.isProduction())
+; 
+
+module.exports = Encore.getWebpackConfig(); 
