diff --git a/public/app.js b/public/app.js
index a2b7e32..dd865ff 100644
--- a/public/app.js
+++ b/public/app.js
@@ -1,5 +1,6 @@
 import Store from './store';
 import './styles.css';
+import sheep from './sheep.png';
 
 const storeEl = document.querySelector('#store');
 const store = new Store(storeEl, [
@@ -11,3 +12,6 @@ const store = new Store(storeEl, [
     'Shearly Conditioned'
 ]);
 store.initialize();
+
+const h1 = document.querySelector('h1');
+h1.innerHTML = `<img src="${sheep}" alt="baa" height="30">` + h1.innerHTML;
