diff --git a/public/app.js b/public/app.js
index 9b83924..c256932 100644
--- a/public/app.js
+++ b/public/app.js
@@ -1,20 +1,4 @@
-class Store {
-    constructor(element, products) {
-        this.products = products;
-        this.element = element;
-    }
-
-    initialize() {
-        const button = document.querySelector('#button');
-        button.addEventListener('click', () => {
-            this.element.innerHTML = this.getProductsString();
-        });
-    }
-
-    getProductsString() {
-        return this.products.join(', ');
-    }
-}
+import Store from './store';
 
 const storeEl = document.querySelector('#store');
 const store = new Store(storeEl, [
diff --git a/public/store.js b/public/store.js
new file mode 100644
index 0000000..a499207
--- /dev/null
+++ b/public/store.js
@@ -0,0 +1,17 @@
+export default class {
+    constructor(element, products) {
+        this.products = products;
+        this.element = element;
+    }
+
+    initialize() {
+        const button = document.querySelector('#button');
+        button.addEventListener('click', () => {
+            this.element.innerHTML = this.getProductsString();
+        });
+    }
+
+    getProductsString() {
+        return this.products.join(', ');
+    }
+}
