diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index 45136e0..74da0da 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -20,7 +20,9 @@ class MainController extends AbstractController
      */
     public function store()
     {
-        return $this->render('main/store.html.twig');
+        return $this->render('main/store.html.twig', [
+            'comments' => ['Great!', 'Super!', 'I love sheep!'],
+        ]);
     }
 
     /**
diff --git a/templates/main/store.html.twig b/templates/main/store.html.twig
index 3d8f6d5..4c6dc53 100644
--- a/templates/main/store.html.twig
+++ b/templates/main/store.html.twig
@@ -16,6 +16,16 @@
             </button>
         </div>
     </div>
+
+    <ul>
+        {% for comment in comments %}
+            <li data-controller="comment">
+                <span class="js-comment-text">{{ comment }}</span>
+
+                <button class="js-btn-make-louder btn btn-sm btn-secondary">Louder</button>
+            </li>
+        {% endfor %}
+    </ul>
 {% endblock %}
 
 {% block javascripts %}
