diff --git a/assets/js/controllers/comment_controller.js b/assets/js/controllers/comment_controller.js
new file mode 100644
index 0000000..14005db
--- /dev/null
+++ b/assets/js/controllers/comment_controller.js
@@ -0,0 +1,11 @@
+import { Controller } from 'stimulus';
+import $ from 'jquery';
+
+export default class extends Controller {
+    initialize() {
+        const $commentText = $(this.element).find('.js-comment-text');
+        $(this.element).find('.js-btn-make-louder').on('click', () => {
+            $commentText.html($commentText.html().toUpperCase());
+        })
+    }
+}
