diff --git a/assets/sheep.png b/assets/images/sheep.png
similarity index 100%
rename from assets/sheep.png
rename to assets/images/sheep.png
diff --git a/assets/app.js b/assets/js/app.js
similarity index 77%
rename from assets/app.js
rename to assets/js/app.js
index 7882a26..5d46e54 100644
--- a/assets/app.js
+++ b/assets/js/app.js
@@ -1,6 +1,6 @@
-import Store from './store';
-import './styles.scss';
-import sheep from './sheep.png';
+import Store from './components/store';
+import '../styles/styles.scss';
+import sheep from '../images/sheep.png';
 
 const storeEl = document.querySelector('#store');
 const store = new Store(storeEl, [
diff --git a/assets/store.js b/assets/js/components/store.js
similarity index 100%
rename from assets/store.js
rename to assets/js/components/store.js
diff --git a/assets/styles.scss b/assets/styles/styles.scss
similarity index 59%
rename from assets/styles.scss
rename to assets/styles/styles.scss
index c1987e2..d68c385 100644
--- a/assets/styles.scss
+++ b/assets/styles/styles.scss
@@ -1,5 +1,5 @@
 #store {
-    background: green url('./sheep.png') repeat top left;
+    background: green url('../images/sheep.png') repeat top left;
     background-size: 30px;
     border-radius: 5px;
     padding: 5px;
diff --git a/webpack.config.js b/webpack.config.js
index 0ca4556..4e53956 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -23,7 +23,7 @@ Encore
      * Each entry will result in one JavaScript file (e.g. app.js)
      * and one CSS file (e.g. app.css) if your JavaScript imports CSS.
      */
-    .addEntry('app', './assets/app.js')
+    .addEntry('app', './assets/js/app.js')
     //.addEntry('page1', './assets/js/page1.js')
     //.addEntry('page2', './assets/js/page2.js')
 
