diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index 2d026bf..45136e0 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -15,6 +15,14 @@ class MainController extends AbstractController
         return $this->render('main/homepage.html.twig');
     }
 
+    /**
+     * @Route("/store", name="app_store")
+     */
+    public function store()
+    {
+        return $this->render('main/store.html.twig');
+    }
+
     /**
      * @Route("/api/products")
      */
diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 208d877..ef3395d 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -3,17 +3,5 @@
 {% block body %}
     <h1>Hello Modern JavaScript!</h1>
 
-    <div class="card">
-        <div class="card-header">
-            Products
-        </div>
-        <div class="card-body">
-            <div class="card-list" id="store"></div>
-
-            <hr>
-            <button id="button" class="btn btn-lg btn-primary">
-                Baaa! <i class="fad fa-sheep"></i>
-            </button>
-        </div>
-    </div>
+    <a href="{{ path('app_store') }}">Go to the store!</a>
 {% endblock %}
diff --git a/templates/main/store.html.twig b/templates/main/store.html.twig
new file mode 100644
index 0000000..01d015b
--- /dev/null
+++ b/templates/main/store.html.twig
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>The Store</h1>
+
+    <div class="card">
+        <div class="card-header">
+            Products
+        </div>
+        <div class="card-body">
+            <div class="card-list" id="store"></div>
+
+            <hr>
+            <button id="button" class="btn btn-lg btn-primary">
+                Baaa! <i class="fad fa-sheep"></i>
+            </button>
+        </div>
+    </div>
+{% endblock %}
+
