diff --git a/templates/main/store.html.twig b/templates/main/store.html.twig
index 01d015b..08310d6 100644
--- a/templates/main/store.html.twig
+++ b/templates/main/store.html.twig
@@ -18,3 +18,14 @@
     </div>
 {% endblock %}
 
+{% block javascripts %}
+    {{ parent() }}
+
+    {{ encore_entry_script_tags('store') }}
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    {{ encore_entry_link_tags('store') }}
+{% endblock %}
