diff --git a/public/app.js b/public/app.js
index cb9e32d..1324f09 100644
--- a/public/app.js
+++ b/public/app.js
@@ -1,5 +1,6 @@
 import Store from './store';
 import Counter from './Counter';
+import axios from 'axios';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
@@ -7,15 +8,13 @@ import ReactDOM from 'react-dom';
 import './styles.css';
 
 const app = document.querySelector('#app');
-const store = new Store(app, [
-    'Sheer Shears',
-    'Wool Hauling Basket',
-    'After-Shear (Fresh Cut Grass)',
-    'After-Shear (Morning Dew)',
-    'Shear Comb',
-    'Shearly Conditioned'
-]);
-store.initialize();
+
+axios.get('/products')
+    .then(response => {
+        const store = new Store(app, response.data);
+        store.initialize();
+    })
+;
 
 ReactDOM.render(
     <Counter />,
diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index 505fc33..de78dc9 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -14,4 +14,19 @@ class MainController extends AbstractController
     {
         return $this->render('main/homepage.html.twig');
     }
+
+    /**
+     * @Route("/products")
+     */
+    public function products()
+    {
+        return $this->json([
+            'Sheer Shears',
+            'Wool Hauling Basket',
+            'After-Shear (Fresh Cut Grass)',
+            'After-Shear (Morning Dew)',
+            'Shear Comb',
+            'Shearly Conditioned',
+        ]);
+    }
 }
