diff --git a/public/app.js b/public/app.js
index 1324f09..6a737a3 100644
--- a/public/app.js
+++ b/public/app.js
@@ -9,12 +9,9 @@ import './styles.css';
 
 const app = document.querySelector('#app');
 
-axios.get('/products')
-    .then(response => {
-        const store = new Store(app, response.data);
-        store.initialize();
-    })
-;
+const response = await axios.get('/products');
+const store = new Store(app, response.data);
+store.initialize();
 
 ReactDOM.render(
     <Counter />,
