diff --git a/public/app.js b/public/app.js
index dd865ff..7882a26 100644
--- a/public/app.js
+++ b/public/app.js
@@ -1,5 +1,5 @@
 import Store from './store';
-import './styles.css';
+import './styles.scss';
 import sheep from './sheep.png';
 
 const storeEl = document.querySelector('#store');
diff --git a/public/styles.css b/public/styles.scss
similarity index 100%
rename from public/styles.css
rename to public/styles.scss
