diff --git a/templates/base.html.twig b/templates/base.html.twig
index baef43c..0c2ff88 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -11,6 +11,8 @@
         <div class="container">
             {% block body %}{% endblock %}
         </div>
-        {% block javascripts %}{% endblock %}
+        {% block javascripts %}
+            <script src="https://kit.fontawesome.com/a20a0555e5.js" crossorigin="anonymous"></script>
+        {% endblock %}
     </body>
 </html>
