diff --git a/public/app.js b/public/app.js
new file mode 100644
index 0000000..593fd08
--- /dev/null
+++ b/public/app.js
@@ -0,0 +1,14 @@
+var products = [
+    'Sheer Shears',
+    'Wool Hauling Basket',
+    'After-Shear (Fresh Cut Grass)',
+    'After-Shear (Morning Dew)',
+    'Shear Comb',
+    'Shearly Conditioned'
+];
+
+var button = document.querySelector('#button');
+var storeEl = document.querySelector('#store');
+button.addEventListener('click', function() {
+    storeEl.innerHTML = products.join(', ');
+});
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 0c2ff88..9fa9009 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -2,7 +2,7 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <title>{% block title %}Welcome!{% endblock %}</title>
+        <title>UK Symfony Meetup: Webpack Encore & A Modern JS Update</title>
         {% block stylesheets %}
             <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
         {% endblock %}
@@ -13,6 +13,7 @@
         </div>
         {% block javascripts %}
             <script src="https://kit.fontawesome.com/a20a0555e5.js" crossorigin="anonymous"></script>
+            <script src="{{ asset('app.js') }}"></script>
         {% endblock %}
     </body>
 </html>
diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 3ebf057..208d877 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -2,4 +2,18 @@
 
 {% block body %}
     <h1>Hello Modern JavaScript!</h1>
+
+    <div class="card">
+        <div class="card-header">
+            Products
+        </div>
+        <div class="card-body">
+            <div class="card-list" id="store"></div>
+
+            <hr>
+            <button id="button" class="btn btn-lg btn-primary">
+                Baaa! <i class="fad fa-sheep"></i>
+            </button>
+        </div>
+    </div>
 {% endblock %}
