diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
new file mode 100644
index 0000000..505fc33
--- /dev/null
+++ b/src/Controller/MainController.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class MainController extends AbstractController
+{
+    /**
+     * @Route("/")
+     */
+    public function homepage()
+    {
+        return $this->render('main/homepage.html.twig');
+    }
+}
diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
new file mode 100644
index 0000000..3ebf057
--- /dev/null
+++ b/templates/main/homepage.html.twig
@@ -0,0 +1,5 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>Hello Modern JavaScript!</h1>
+{% endblock %}
