diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index 505fc33..2d026bf 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -14,4 +14,19 @@ class MainController extends AbstractController
     {
         return $this->render('main/homepage.html.twig');
     }
+
+    /**
+     * @Route("/api/products")
+     */
+    public function productsApi()
+    {
+        return $this->json(['products' => [
+            'Sheer Shears',
+            'Wool Hauling Basket',
+            'After-Shear (Fresh Cut Grass)',
+            'After-Shear (Morning Dew)',
+            'Shear Comb',
+            'Shearly Conditioned'
+        ]]);
+    }
 }
