diff --git a/public/app.js b/public/app.js
index cace249..34ec205 100644
--- a/public/app.js
+++ b/public/app.js
@@ -17,12 +17,14 @@ application.load(definitionsFromContext(context))
 
 const app = document.querySelector('#app');
 
+/*
 async function initializeStore() {
     const response = await axios.get('/products');
     const store = new Store(app, response.data);
     store.initialize();
 }
 initializeStore();
+*/
 
 ReactDOM.render(
     <Counter />,
