diff --git a/public/app.js b/public/app.js
index 30062e6..cace249 100644
--- a/public/app.js
+++ b/public/app.js
@@ -7,6 +7,14 @@ import ReactDOM from 'react-dom';
 // Could this somehow load a CSS file for us?
 import './styles.css';
 
+/* Stimulus Setup */
+import { Application } from 'stimulus'
+import { definitionsFromContext } from 'stimulus/webpack-helpers'
+const application = Application.start()
+const context = require.context('./controllers', true, /\.js$/)
+application.load(definitionsFromContext(context))
+/* End Stimulus Setup */
+
 const app = document.querySelector('#app');
 
 async function initializeStore() {
