diff --git a/public/controllers/store_controller.js b/public/controllers/store_controller.js
index dbe4209..51c6149 100644
--- a/public/controllers/store_controller.js
+++ b/public/controllers/store_controller.js
@@ -1,5 +1,11 @@
 import { Controller } from 'stimulus';
 
 export default class extends Controller {
-    
+    static get targets() {
+        return ['productList'];
+    }
+
+    printProducts() {
+        this.productListTarget.innerHTML = 'Testing!'
+    }
 }
diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 463845c..ef94bf2 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -3,15 +3,18 @@
 {% block body %}
     <h1>Hello Modern JavaScript!</h1>
 
-    <div class="card">
+    <div class="card" data-controller="store">
         <div class="card-header">
             Products
         </div>
         <div class="card-body">
-            <div class="card-list" id="app"></div>
+            <div class="card-list" data-target="store.productList"></div>
 
             <hr>
-            <button id="button" class="btn btn-lg btn-primary">
+            <button
+                data-action="click->store#printProducts"
+                class="btn btn-lg btn-primary"
+            >
                 Baaa! <i class="fad fa-sheep"></i>
             </button>
         </div>
