diff --git a/assets/js/controllers/store_controller.js b/assets/js/controllers/store_controller.js
index 51f2a8b..714663c 100644
--- a/assets/js/controllers/store_controller.js
+++ b/assets/js/controllers/store_controller.js
@@ -2,13 +2,17 @@ import { Controller } from 'stimulus';
 import axios from 'axios';
 
 export default class extends Controller {
+    static get targets() {
+        return ['productList']
+    }
+
     async initialize() {
         const response = await axios.get('/api/products');
         this.products = response.data.products;
     }
 
     showProducts() {
-        this.element.querySelector('#store')
+        this.productListTarget
             .innerHTML = this.getProductsString();
     }
 
diff --git a/assets/styles/styles.scss b/assets/styles/styles.scss
index d68c385..3f4254b 100644
--- a/assets/styles/styles.scss
+++ b/assets/styles/styles.scss
@@ -1,4 +1,4 @@
-#store {
+.card-list {
     background: green url('../images/sheep.png') repeat top left;
     background-size: 30px;
     border-radius: 5px;
diff --git a/templates/main/store.html.twig b/templates/main/store.html.twig
index 5794ef3..3d8f6d5 100644
--- a/templates/main/store.html.twig
+++ b/templates/main/store.html.twig
@@ -8,7 +8,7 @@
             Products
         </div>
         <div class="card-body">
-            <div class="card-list" id="store"></div>
+            <div class="card-list" data-target="store.productList"></div>
 
             <hr>
             <button data-action="click->store#showProducts" class="btn btn-lg btn-primary">
