diff --git a/assets/js/controllers/store_controller.js b/assets/js/controllers/store_controller.js
index 4bd416e..51f2a8b 100644
--- a/assets/js/controllers/store_controller.js
+++ b/assets/js/controllers/store_controller.js
@@ -1,13 +1,18 @@
 import { Controller } from 'stimulus';
 import axios from 'axios';
-import Store from '../components/store';
 
 export default class extends Controller {
     async initialize() {
         const response = await axios.get('/api/products');
-        const products = response.data.products;
+        this.products = response.data.products;
+    }
+
+    showProducts() {
+        this.element.querySelector('#store')
+            .innerHTML = this.getProductsString();
+    }
 
-        const store = new Store(this.element, products);
-        store.initialize();
+    getProductsString() {
+        return this.products.join(', ');
     }
 }
diff --git a/templates/main/store.html.twig b/templates/main/store.html.twig
index 1c67925..5794ef3 100644
--- a/templates/main/store.html.twig
+++ b/templates/main/store.html.twig
@@ -11,7 +11,7 @@
             <div class="card-list" id="store"></div>
 
             <hr>
-            <button id="button" class="btn btn-lg btn-primary">
+            <button data-action="click->store#showProducts" class="btn btn-lg btn-primary">
                 Baaa! <i class="fad fa-sheep"></i>
             </button>
         </div>
