diff --git a/webpack.config.js b/webpack.config.js
index 8d66465..d1053c6 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -8,5 +8,14 @@ module.exports = {
     output: {
         path: path.resolve(__dirname, 'public', 'build'),
     },
-    mode: 'development'
+    mode: 'development',
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                loader: 'babel-loader'
+            }
+        ]
+    }
 }
